tests/subsys/bluetooth/controller/crypto/mock/ble_controller_soc_mock.h

```diff
@@ -0,0 +1,20 @@
+
```

Comment:
Do you need to add this header file? I guess it should be sufficient to include ble_controller_soc.h?